src/main/java/org/variantsync/diffdetective/diff/difftree/parse/LogicalLine.java

```diff
@@ -0,0 +1,80 @@
+package org.variantsync.diffdetective.diff.difftree.parse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.variantsync.diffdetective.diff.DiffLineNumber;
+import org.variantsync.diffdetective.util.Assert;
+
+/**
+ * A Logical line consisting of multiple physical lines joined by line continuations.
```

Comment:
Please add `@author` and elaborate on the meaning of `line`. The word `line` is more general than lines within text-based files as we inspect.

Reply:
Good catch. I didn't add `@author` anywhere. This tag is now included in every file I created.
Whats your policy for changing `@author`? For example the `DiffTreeParser` doesn't have an `@author` annotation. Should I add one. If yes, who should be mentioned?

I didn't quite get the second part. What other meanings of line are there?
The logical and physical part is now explained by referencing the C11 standard and providing a regex for more rigorous explanation. What else should I mention?

Reply:
Ok thank you. I guess I forgot to add the author tag at some places too. Just leave these places for now.

I was referring to something like "line of text" or "line of code" because a just a line is also something that you can draw with a pencil. :D

Reply:
Oh, now I get it. It makes sense that not everybody associates logical and physical lines with source code :D.
I added that physical lines are
> of a text file

to the summary of the class.